scripts/build-tools.sh

```diff
@@ -4,6 +4,7 @@
 
 # fail immediately on any errors
 set -e
+set -x
 
 print_usage()
```

Comment:
This commit should be removed right?

Reply:
Yes, here because trying to debug the failure, will remove before final review. 

> I think this build fails because of version difference and there's no -S option in that version. What version of cmake are we trying to support that I should be using as reference for commands?

From my previous comment ^^

Reply:
@crazoes don't know to answer your questions. Maybe @jajanusz can jump in.

Anyway, why does the build fail now and it worked without this patch? Will make some time later to investigate but this shouldn't happen.

Reply:
Yes, it works locally. cmake version I wrote this against is 3.17.3

Reply:
From the top of `sof/CMakeLists.txt`:

```
cmake_minimum_required(VERSION 3.10)
```
3.10 is 2.5 years old which is not crazy old. On the other hand there are many repositories and other ways to upgrade cmake easily, so I think we should bump that to 3.13 which has a few useful features like for instance `-B`. We should check the default version in most popular Linux distributions. Off-topic sorry.